src/templates/static/_layout/footer.jsx

```diff
@@ -199,66 +199,70 @@ const FooterNormal = () => (
+                                        <img id='gamstop_icon_footer' className='responsive' src={it.url_for('images/pages/footer/gamstop.svg')} />
                                     </a>
-                                    <div data-show='iom, malta, default' className='id-hide'>
-                                        <a href='https://www.gamstop.co.uk' target='_blank' rel='noopener noreferrer'>
-                                            <img id='gamstop_icon_footer' className='responsive' src={it.url_for('images/pages/footer/gamstop.svg')} />
-                                        </a>
-                                    </div>
                                 </div>
                             </div>
+                            <SocialIcons
+                                networks={[
+                                    { media: 'youtube',     href: 'https://www.youtube.com/user/BinaryTradingVideos' },
+                                    { media: 'google-plus', href: 'https://plus.google.com/106251151552682209951' },
+                                    { media: 'facebook',    href: 'https://www.facebook.com/binarydotcom' },
+                                    { media: 'twitter',     href: 'https://twitter.com/Binarydotcom' },
+                                    { media: 'telegram',    href: 'https://t.me/binarydotcom' },
+                                    { media: 'reddit',      href: 'https://www.reddit.com/r/binarydotcom/' },
+                                ]}
+                            />
                         </div>
-
-                        <SocialIcons
-                            is_centered
-                            networks={[
-                                { media: 'youtube',     href: 'https://www.youtube.com/user/BinaryTradingVideos' },
-                                { media: 'google-plus', href: 'https://plus.google.com/106251151552682209951' },
-                                { media: 'facebook',    href: 'https://www.facebook.com/binarydotcom' },
-                                { media: 'twitter',     href: 'https://twitter.com/Binarydotcom' },
-                                { media: 'telegram',    href: 'https://t.me/binarydotcom' },
-                                { media: 'reddit',      href: 'https://www.reddit.com/r/binarydotcom/' },
-                            ]}
-                        />
                     </div>
                 </div>
-            </div>
-        </div>
-        <div id='footer-last' className='primary-bg-color invisible'>
-            <div className='container'>
-                <div className='gr-padding-10'>
-                    <p className='gambling content-inverse-color no-para-margin'>
-                        {it.L('[_1] offers complete trading solutions for binary options[_2] and CFDs[_3].', it.website_name, '<span class="eu-only invisible">', '</span>')}
```

Comment:
@ashkanx do we remove this now? i remember this was added in the recent footer updates

Reply:
There is a new design for Footer. you can copmare against that